com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -267,28 +269,21 @@ private void OnValidate()
                 }
             }
 
-            var activeScene = UnityEngine.SceneManagement.SceneManager.GetActiveScene();
-            var activeSceneName = activeScene.name;
-            if (!NetworkConfig.RegisteredScenes.Contains(activeSceneName))
+            if (NetworkConfig.EnableSceneManagement)
             {
-                if (NetworkLog.CurrentLogLevel <= LogLevel.Normal)
+                foreach(var sceneAsset in NetworkConfig.RegisteredSceneAssets)
                 {
-                    NetworkLog.LogWarning("Active scene is not registered as a network scene. Netcode has added it");
-                }
-
-                NetworkConfig.RegisteredScenes.Add(activeSceneName);
-                UnityEditor.EditorApplication.delayCall += () =>
-                {
-                    if (!UnityEditor.EditorApplication.isPlaying)
+                    if(!NetworkConfig.RegisteredScenes.Contains(sceneAsset.name))
                     {
-                        UnityEditor.EditorUtility.SetDirty(this);
-                        UnityEditor.SceneManagement.EditorSceneManager.MarkSceneDirty(activeScene);
+                        NetworkConfig.RegisteredScenes.Add(sceneAsset.name);
                     }
-                };
+                }
             }
 
+            var activeScene = UnityEngine.SceneManagement.SceneManager.GetActiveScene();
+
             // If the scene is not dirty or the asset database is currently updating then we can skip updating the NetworkPrefab information
-            if (!activeScene.isDirty || UnityEditor.EditorApplication.isUpdating)
+            if (!activeScene.isDirty || EditorApplication.isUpdating)
```

Comment:
It will not be included if we are not in the editor.
Scroll up to line 256 and you will see the entire OnValidate method is wrapped in a #if UNITY_EDITOR...#endif.

So, this is nothing to worry about.

Reply:
Ah nice I missed that!